config/routes.rb

```diff
@@ -75,14 +75,11 @@
   namespace :admin do
     get 'index', path: '/'
 
-    resources :report_generator do
+    resources :report_generator, only: [:index] do
```

Comment:
@tahnok Does this look better?
![2017-07-06_133607](https://user-images.githubusercontent.com/22206278/27924698-264bd482-6250-11e7-9cd4-cdb3ca3e0b26.jpg)